python/pyspark/sql/tests.py

```diff
@@ -3036,6 +3052,9 @@ def test_toPandas_arrow_toggle(self):
         pdf = df.toPandas()
         self.spark.conf.set("spark.sql.execution.arrow.enable", "true")
```

Comment:
It's meant to be enabled for the duration of running `ArrowTests` (except for 1 test that disables then enables again).  The `SparkSession` is stopped in `tearDownClass` so it will not affect any other tests right?